homeassistant/components/notify/homematic.py

```diff
@@ -52,23 +52,10 @@ def __init__(self, hass, data):
 
     def send_message(self, message="", **kwargs):
         """Send a notification to the device."""
-        attr_data = kwargs.get(ATTR_DATA)
-        if attr_data is not None:
-            if 'address' in attr_data:
-                self.data[ATTR_ADDRESS] = attr_data['address']
-            if 'channel' in attr_data:
-                self.data[ATTR_CHANNEL] = attr_data['channel']
-            if 'param' in attr_data:
-                self.data[ATTR_PARAM] = attr_data['param']
-            if 'value' in attr_data:
-                self.data[ATTR_VALUE] = attr_data['value']
-            if 'interface' in attr_data:
-                self.data[ATTR_INTERFACE] = attr_data['interface']
+        data = {self.data, kwargs.get(ATTR_DATA, {})}
```

Comment:
This doesn't look correct. Probably use `dict.update`.

Reply:
that is correct, but work only with python 3.6 :(

Reply:
Works in 3.5 but you need some `**`

![image](https://user-images.githubusercontent.com/1444314/46870429-dd373380-ce2e-11e8-873b-f7e5f729477e.png)